src/dynsight/_internal/vision/video_to_frame.py

```diff
@@ -8,99 +8,79 @@
 if TYPE_CHECKING:
     import pathlib
 
-    import numpy as np
-
 
 @dataclass
 class Video:
-    """Load a video file and provides utilities.
+    """Load a video file and provide utilities.
+
+    This class loads a video from a file path and provides methods
+    to retrieve video information and extract frames.
+
+    * Author: Simone Martino
+
+    .. caution::
+        This part of the code is still under development and may
+        contain errors.
+
+    Parameters:
+        video_path: Path to the video file.
 
-    Attributes:
-        video_path:
-            File path to the video.
     """
 
     video_path: pathlib.Path
-    frames: list[np.ndarray] = field(default_factory=list)
+    """Path to the video file."""
```

Comment:
See my comment above about attributes and how to comment them in a way that shows on the docs.

Reply:
Finish this.